docs/release-process/new-release-branch.md

```diff
@@ -0,0 +1,55 @@
+# Creating a new release branch
+
+To release any given major.minor version of Go, the microsoft/go repository needs to have a branch tracking that version.
+Early minor/patch/prerelease versions are likely to not have a branch yet, e.g. 1.19, 1.19beta1, and 1.19rc1.
+
+You can check for a specific release branch here: [list of all microsoft/go branches beginning in microsoft/release-branch.go](https://github.com/microsoft/go/branches/all?query=microsoft%2Frelease-branch.go).
+
+> In Go versions prior to 1.19, you may notice an additional `microsoft/dev.boringcrypto.go*` branch.
+> This is no longer necessary in Go 1.19+.
+
+If there isn't an existing branch, you need to create one.
+The steps below use `1.x` as a stand-in for the target version of Go and `main` for the version built by `microsoft/main`:
```

Comment:
`microsoft/main` is not introduced anywhere. Do you mean `microsoft/go`? Same applies for a couple of appearances in the next paragraph.

Reply:
I mean the branch, but yeah, needs to be clearer since this kind of name is a repo name most of the time in these docs.